test/jdk/jdk/internal/vm/Continuation/Fuzz.java

```diff
@@ -91,6 +91,9 @@ public static void main(String[] args) {
             throw new SkippedException("Test is unstable with slowdebug bits "
                                        + "on macosx-aarch64");
         }
+        if (Platform.isLinux() && Platform.isPPC()) {
```

Comment:
I wouldn't limit to Linux.

Reply:
I changed it, removed the Linux related check.